aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,69 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## April 2023
+
+- **Brand Refresh**
+  - Figma scoping has been completed.
+  - Designers began updating theme in Figma.
+  - [New brand refresh is now available on Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=0%3A1&t=ViKCtdnRM2CUkNZI-1)!
+  - To view details on how to solve for common Figma problems or to migrate to the new theme, check out the [Figma migration guide](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2%3A4&t=ViKCtdnRM2CUkNZI-1).
+  - Designers have been setting up calls with different service teams to partner and strategize on how they may begin to migrate to the new theme.
+  - [Wizard titles](https://design-system.hpe.design/templates/wizard) have gone from bold to medium font weight.
+- **New Global Header**
+  - The team has been working with the HFS team to prototype the new Global Header.
+- **Figma Component Updates**
+ - Figma components have been updated to use the correct theme colors for placeholder and disabled text. Previously, the placeholder text was xweak. It has been updated to use the correct color, text-weak to meet accessibility. The following components have been updated:
+    - [Checkbox](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2940%3A62424&t=ViKCtdnRM2CUkNZI-1)
+    - [CheckboxGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5732%3A78041&t=ViKCtdnRM2CUkNZI-1)
+    - [RadioButton](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=3520%3A83485&t=ViKCtdnRM2CUkNZI-1)
+    - [RadioButtonGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5733%3A92878&t=ViKCtdnRM2CUkNZI-1)
+    - [TextInput](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2%3A8&t=ViKCtdnRM2CUkNZI-1)
+    - [Layer](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5735%3A85900&t=ViKCtdnRM2CUkNZI-1)  
+- **FileInput**
+  - We have added the alert icon within [FileInput in Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=387%3A1967&t=ViKCtdnRM2CUkNZI-1).
+- **Data**
+  - [Data Filters now have a tooltip](https://design-system.hpe.design/templates/filtering#filtering-with-selectable-results) when hovering over the filter button.
+
+## March 2023
+
+- **Data**
+  - Refactored [Data on DataTable Customization on site](https://design-system.hpe.design/templates/datatable-customization)
```

Comment:
If someone outside of our team were reading this I think it would be hard to understand what was changed and why they should care. 'Refactored' is a bit vague.
```suggestion
  - Changed [DataTable Customization example](https://design-system.hpe.design/templates/datatable-customization) to use the new Data components.
```